dev-docs/Cloud-Env-Testing.md

```diff
@@ -47,19 +43,24 @@ Follow these steps to run the workflow:
       e.g. `elastic / unified-release - staging # 8.9 - 11 - 8.9.0-c6bb8f7a Success after 4 hr 58 min`. Now just copy it
       and replace it the image path: `docker.elastic.co/cloud-release/elastic-agent-cloud:8.9.0-c6bb8f7a`.
 
-    - `run-sanity-tests` (optional): Set to `true` to run sanity tests after the environment is set up. Default: `false`
+    - **`run-sanity-tests`** (**optional**): Set to `true` to run sanity tests after the environment is set up. Default: `false`
 
-    - `cleanup-env` (optional): Set to `true` if you want the resources to automatically be cleaned up after
+    - **`cleanup-env`** (**optional**): Set to `true` if you want the resources to automatically be cleaned up after
       provisioning - useful if you don't want to test the env manually after deployment.
       Default: `false`.
 
-      .
-   ![Adjust Inputs](https://github.com/elastic/cloudbeat/assets/99176494/bac5004d-7cbc-4a34-8127-3acd11acc90e)
+    - **`ec-api-key`** (**optional**): By default, all the new environments will be created in our EC Cloud Security organization.
+      If you want to create the environment on your personal org (`@elastic.co`) you can enter
+      your private [Elastic Cloud](https://cloud.elastic.co/home) API key. Follow the
+      [Cloud API Keys](https://www.elastic.co/guide/en/cloud/current/ec-api-authentication.html) documentation for
+      step-by-step instructions on generating the token.
 
+   ![Optional Parameters](https://github.com/oren-zohar/cloudbeat/assets/85433724/17933589-ee0e-4181-a244-f501f54bda6c)
```

Comment:
nit: We may need to upload images to the https://github.com/elastic/cloudbeat repo